src/core/client.ts

```diff
@@ -216,15 +239,16 @@ export default class Client {
         class: notice.name,
         message: notice.message,
         backtrace: notice.backtrace,
-        fingerprint: notice.fingerprint
+        fingerprint: notice.fingerprint,
+        tags: notice.tags
       },
       request: {
         url: filterUrl(notice.url, this.config.filters),
         component: notice.component,
         action: notice.action,
         context: notice.context,
         cgi_data: cgiData,
-        params: filter(notice.params, this.config.filters) || {},
+        params: filter(notice.params, this.config.filters),
```

Comment:
Does this still need the fallback?

```suggestion
        params: filter(notice.params, this.config.filters) || {},
```

Reply:
Good catch. Forgot to put it back since I was making an object.